src/content/docs/instrumentation-editor-instrument-net-ui.mdx

```diff
@@ -1,4 +1,4 @@
----
+localhost:8000/docs/instrumentation-editor-instrument-net-ui---
```

Comment:
Why do we need this?

Reply:
Good catch, thank you. I was testing and I imagine that I paste the wrong url into the wrong page. Thank you for letting me know!